pep-0480.txt

```diff
@@ -341,14 +345,15 @@ distributions, and prevents MITM attacks on usernames and passwords.
 __ https://github.com/pypa/twine
 
 
-Distutils
----------
+Setuptools
+----------
 
-`Distutils`__ MAY be modified to sign metadata and to upload signed distributions
-to PyPI.  Distutils comes packaged with CPython and is the most widely-used
-tool for uploading distributions to PyPI.
+`Setuptools`__ MAY be modified to sign metadata and to upload signed
+distributions to PyPI.  Setuptools is a library which implements
+low-level functions that relate to packaging and publication of
+Python software.
```

Comment:
Given that this is no longer referencing distutils (which had a privileged position as being in the stdlib) would it be more reasonable to talk about "build backends" *in general* here? In particular, setuptools is *not* a "library which implements low-level functions" - it's a build backend like many others, and whereas other backends could reasonably depend on distutils for signing functions, they would *not* be likely to want a dependency on setuptools.

Reply:
Thanks. Fixed.